jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java

```diff
@@ -1,36 +1,17 @@
 package io.github.com.pages.sections;
 
-import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.angular.elements.complex.RadioButtons;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
-import com.epam.jdi.light.ui.html.elements.common.Button;
 import com.epam.jdi.light.ui.html.elements.common.Text;
-import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
 
 public class RadioButtonSection {
-    @Css("#basic-radio-group")
-    public static RadioButtons basicRadioButtons;
-
-    @UI("#radio-option-one")
-    public static Button firstBasicRadioButton;
 
-    @UI("#radio-option-two")
-    public static Button secondBasicRadioButton;
+    @UI("#basic-radio-group")
```

Comment:
The annotation @UI("#basic-radio-group") is not required here.
The "smart locator" is working properly.